### DISCUSSION TITLE
Any plan for es format of @swc/core?

### DISCUSSION THREAD

**Discussion by aelbore** [swc-project/swc#2062] [Q&A]
current version of @swc/core is on commonjs format it would be good have a es format/version of it, i started working of my new projects that uses swc to have es format especially im using the Visitor.js

**Comment by kdy1** [comment]
As it's a breaking change, it will be only possible with v2.

**Reply by aelbore** [reply]
@kdy1 thanks will look forward into this

i think we can use hybrid, it will be 2 formats, commonjs and es format using conditional "exports" in package.json
